.github/workflows/tag-release.yml

```diff
@@ -43,9 +43,9 @@ jobs:
           token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
 
       - run: yarn
-      - name: Tag prerelease
+      - name: Tag release
         run: |
           # setup the username and email.
           git config --global user.name "Budibase Staging Release Bot"
           git config --global user.email "<>"
-          ./scripts/versionCommit.sh ${{ github.event.inputs.versioning }}
+          ./scripts/versionCommit.sh ${{ github.event.inputs.versioning:-patch }}
```

Comment:
Defaulting to a patch release when pushing to master, as it was done before

Reply:
perfect 👍